storage/rocksdb/ha_rocksdb.cc

```diff
@@ -11971,6 +11971,7 @@ void rocksdb_set_update_cf_options(THD *const /* unused */,
     // NO_LINT_DEBUG
     sql_print_warning("MyRocks: NULL is not a valid option for updates to "
```

Comment:
If `NULL` is not meant to be a valid option, a better solution may be to add a check callback function that checks if `val` is null and fail the set command.

Reply:
I'm not that confident it is cleaner because NULL is the default value. But I don't feel strongly about it and the code works, so will update the PR as suggested.

Reply:
I think we should change default value to "" then. I'm looking other similar variables like `default_cf_options` and `override_cf_options` and both default to "", not nullptr.

Reply:
Sounds good to me.